configure.ac

```diff
@@ -703,6 +703,8 @@ AC_CHECK_FUNCS([mblen], [], [HAVE_MBLEN=1], [])
 
 AC_CONFIG_FILES([Makefile
 		 man/ctags.1.rst
+		 man/ctags-incompatibilities.1.rst
+		 man/ctags-optlib.1.rst
```

Comment:
Should ctags-incompatibilities and ctags-optlib be in the section 1?
Isn't it better to put in the section 7?
https://en.wikipedia.org/wiki/Man_page#Manual_sections

It's also better to mention these pages in the "SEE ALSO" subsection.